libvast/vast/ewah_bitmap.hpp

```diff
@@ -102,6 +102,8 @@ class ewah_bitmap : public bitmap_base<ewah_bitmap>,
 
   size_type size() const;
 
+  size_t memusage() const;
```

Comment:
If I recall correctly, then we use `num_bytes` in some places, `size_bytes` in others, and now also `memusage`. Can we consolidate these?

Reply:
`num_bytes` is used for different purposes, but `size_bytes()` seems to fulfill the same role as `memusage()`. From my perspective `memusage()` is a more descriptive name for the intent of the function. So I will rename `size_bytes()`.

Reply:
Fine by me.